handlers/block.go

```diff
@@ -117,13 +117,14 @@ func Block(w http.ResponseWriter, r *http.Request) {
 		}
 		data.Data = futurePageData
 
-		if handleTemplateError(w, r, blockFutureTemplate.ExecuteTemplate(w, "layout", data)) != nil {
+		if handleTemplateError(w, r, "block.go / Block / ErrNoRows", blockFutureTemplate.ExecuteTemplate(w, "layout", data)) != nil {
 			return // an error has occurred and was processed
 		}
 		return
 	} else if err != nil {
-		logger.Errorf("error executing template for %v route: %v", r.URL.String(), err)
-		http.Error(w, "Internal server error", http.StatusInternalServerError)
+		if handleTemplateError(w, r, "block.go / Block / GetSlotPageData", err) != nil {
+			return // an error has occurred and was processed
+		}
```

Comment:
The http error code returned has changed from `http.StatusInternalServerError` to `http.StatusServiceUnavailable`.
Is this change intentional?